### DISCUSSION TITLE
App name use Chinese,toga demo,app icon customized....

### DISCUSSION THREAD

**Discussion by guaiguaicai** [beeware/briefcase#611] [Q&A]
Describe the bug
Question 1:
When using the briefcase new command, the App wants to use the Chinese name, but an error occurs:
Question 2:
The title in self.main_window = toga.MainWindow(title=self.formal_name) is customized, such as "hello", there is no effect when the Android App is running
Question 3:
Can the App icon be customized, and if so, where is the tutorial?
Question 4:
Where can there be more detailed grammar instructions or demos for toga? The online documentation says it is too simple to operate. For example, toga.MainWindow's info_dialog, I don't know how to use it if I want to use it.

Thank you very much for making such a good software as beeware, which can make Python have infinite possibilities, thank you!

To Reproduce
Formal Name [Hello World]: 学口算
Traceback (most recent call last):
File "D:\Python\Python38\lib\runpy.py", line 194, in _run_module_as_main
return run_code(code, main_globals, None,
File "D:\Python\Python38\lib\runpy.py", line 87, in run_code
exec(code, run_globals)
File "D:\Python\Python38\envs\beeware\Scripts\briefcase.exe_main.py", line 7, in
File "d:\python\python38\envs\beeware\lib\site-packages\briefcase_main.py", line 11, in main
command(**options)
File "d:\python\python38\envs\beeware\lib\site-packages\briefcase\commands\new.py", line 523, in call
state = self.new_app(template=template, **options)
File "d:\python\python38\envs\beeware\lib\site-packages\briefcase\commands\new.py", line 461, in new_app
context = self.build_app_context()
File "d:\python\python38\envs\beeware\lib\site-packages\briefcase\commands\new.py", line 320, in build_app_context
class_name = self.make_class_name(formal_name)
File "d:\python\python38\envs\beeware\lib\site-packages\briefcase\commands\new.py", line 84, in make_class_name
if class_name[0].isdigit():
IndexError: string index out of range

Expected behavior

Screenshots

Environment:

Operating System: win10
Python version: 3.8
Software versions:
Briefcase: 0.3.5
Toga:
...
Additional context
In English mode, I can use beeware normally. But I have difficulty in using beeware personally, and I want the help of detailed documentation and demo
Or where there are learning exchanges and places, there are too few materials to find

I also asked a question under beeware. I asked the wrong place. Please delete that question, thank you!

Thank you very much for making such a good software as beeware, which can make Python have infinite possibilities, thank you!

**Comment by freakboy3742** [comment]
1 looks like a bug, caused by not testing non-Latin app names. That should be a relatively simple thing to fix if you want to try contributing a fix.

2 also looks like a bug; the Android app is taking it's window title from the *app*, not the MainWindow. This won't be *as* simple to fix, but shouldn't be *too* hard.

The App icon *can* be customised, using the [`icon` setting in your project file](https://briefcase.readthedocs.io/en/latest/reference/configuration.html#icon), as well as some [Android specific requirements](https://briefcase.readthedocs.io/en/latest/reference/platforms/android.html). If you would like an example of what this looks like in practice [this project](https://github.com/freakboy3742/traveltips) may be helpful.

As for other documentation - all the available documentation for toga is at https://toga.readthedocs.io. There's definitely a need for a lot more documentation - as a project, we're still at v0.3, and documentation is one of the things we know we need to improve.

**Comment by guaiguaicai** [comment]
Formal Name cannot contain the symbol '' when assigned, and there is an error compiling, which is a bug.

Second, I'm still learning python, and I'm honored to be able to patch bugs for beeware, but I don't know how to fix them.
Icon customization I will try to use, thank you for your advice.

**Comment by freakboy3742** [comment]
I've opened #612 to track the first problem you describe; and #613 to track the second.
